### PR TITLE
Task 8 ready for review

### PR DESCRIPTION
**Pull request by semkaznacheev**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -15,6 +15,8 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@500&display=swap" rel="stylesheet">
+    <style> @import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@500&display=swap'); </style>
```
Comment (k-huggs) [inline public/index.html]: You should do this all in your stylesheet where the rest of your styles are taking place

File: src/components/Comments.jsx [hunk 2]
```diff
@@ -1,42 +1,113 @@
...
+    .then(() => {
+        
+        alert("Your comment was posted. Congrats!")
+    })
+    .catch(() => {
+        setErr("Somethimg went wrong. Check your username and fill out all fields")
+    })
+}
+
+const usernameHandler = (event) => {
```
Comment (k-huggs) [inline src/components/Comments.jsx]: remove the need to type a username to leave a comment. You can either hard code it, or make that you can only post a comment as a logged in user. This makes it easier to navigate

File: src/components/Comments.jsx [hunk 3]
```diff
@@ -1,42 +1,113 @@
...
 const {article_id} = props;
 const [isLoading, setIsLoading] = useState(true);
-const [comments, setComments] = useState([])
+const [comments, setComments] = useState([]);
+const [username, setUserName] = useState('');
+const [body, setBody] = useState('');
+const [comment, setComment] = useState({});
+const [err, setErr] = useState(null);
+
+const submitHandler = (event) => {
```
Comment (k-huggs) [inline src/components/Comments.jsx:15]: after successfully posting a comment, clear the input box

**Review by k-huggs — COMMENTED** [review]
Hi mate, a couple of changes to be made, please see the comments on the code.

Your app isn't fully responsive, so try to fix this as you go along.

File: public/index.html (hunk 1 above)
Comment (semkaznacheev) [reply public/index.html]: Hi, Kyle. Done. Thanks

File: src/components/Comments.jsx (hunk 2 above)
Comment (semkaznacheev) [reply src/components/Comments.jsx]: Done

File: src/components/Comments.jsx (hunk 3 above)
Comment (semkaznacheev) [reply src/components/Comments.jsx:15]: Done

**Review by k-huggs — COMMENTED** [review]
All good!

**Pull request merged**